build/KoreBuild.sh

```diff
@@ -152,7 +152,13 @@ export ReferenceAssemblyRoot=$NUGET_PACKAGES/netframeworkreferenceassemblies/$ne
 makeFileProj="$scriptRoot/KoreBuild.proj"
 msbuildArtifactsDir="$repoFolder/artifacts/msbuild"
 msbuildResponseFile="$msbuildArtifactsDir/msbuild.rsp"
-msbuildLogFile="$msbuildArtifactsDir/msbuild.binlog"
+msBuildLogArgument=""
+
+if [ ! -z "$KOREBUILD_ENABLE_BINARY_LOG" ]; then
```

Comment:
`KOREBUILD_SKIP_RUNTIME_INSTALL` is bizarrely inconsistent. The rest of this script uses e.g. `if ($env:KOREBUILD_DOTNET_CHANNEL)` for PowerShell and the matching `[ -z "$KOREBUILD_DOTNET_CHANNEL" ] &&` for `bash`.